e2e/settings.spec.ts

```diff
@@ -1,4 +1,4 @@
-import test from "@playwright/test";
+import { test, expect } from "@playwright/test";
 import { loggedInAsUserOne } from "./utils";
 
 test.describe("Unauthenticated setttings Page", () => {
```

Comment:
Can we change this to authenticated settings page. Because the user is authenticated for these tests

Reply:
I'm not sure exactly what you're saying here? 

Did you want a separate file for auth vs. un-auth tests, because doesn't a user have to be logged in to edit their profile?

I guess I could use more detail on this 👌🏻

Reply:
Your new test is ran inside a test suite called 'Unauthenticated setttings Page'

But this is an authenticated test because users need to login to change their username.

If you just change the name of the test it will be fixed